pages/dkp/kommander/1.4/clusters/attach-cluster/cluster-with-networking-restrictions/index.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: layout.pug
+navigationTitle: Attach a cluster with no networking restrictions
+title: Attach a cluster with no networking restrictions
+menuWeight: 10
+excerpt: How to attach an existing cluster that has no additional networking restrictions
+beta: false
+---
+
+Use this option when you want to attach a cluster that is in a DMZ, behind a proxy server or a firewall, or that requires additional access information. This procedure gathers the information required to create a kubeconfig file for the netork tunnel between Kommander and the cluster you want to attach.
+
+1. Select the **Attach Cluster** option in the **Actions** dopdown menu at the top right to display the **Connection Information** dialog box.
+
+1. Select the card, Attach Cluster with Networking Restrictions to display the configuration page.
```

Comment:
Should the "Attach Cluster with Networking Restrictions" text be highlighted like the other UX elements are in this doc?

```suggestion
1. Select the **Attach Cluster with Networking Restrictions** card to display the configuration page.
```
??

Reply:
Missed this last night. Fixed under D2IQ-74942 on 4/8/21.